storage/src/main/java/org/apache/uniffle/storage/handler/impl/MemoryClientReadHandler.java

```diff
@@ -86,7 +108,7 @@ public ShuffleDataResult readShuffleData() {
         partitionId,
         lastBlockId,
         readBufferSize,
-        expectedTaskIdsBitmapFilterEnable ? expectTaskIds : null
```

Comment:
Why do we remove `expectedTaskIdsBitmapFilterEnable`?

Reply:
If expectedTaskIdsBitmapFilterEnable=false, expectTaskIds will be null